README.md

```diff
@@ -51,6 +50,48 @@ blocked:000000
 bug:fc2929
 ```
 
+#### Deleting Labels
+
+Sometimes you might want to delete some labels (likely Github's default labels) in the process of setting up a repo with your label config.
+
+You can add in the your label config file a line with `-delete:tag-name` to go and delete a specific label.
+
+For example:
+
+```
+-delete:wontfix
```

Comment:
We could add a custom label config that is only responsible for clearing gh labels.

`clear-default-labels`

```
-delete:wontfix
-delete:question
...
```

Then from a labels config, we could call:

```
-import:clear-default-labels
```

While this use case I imagine is a very popular use case, I would rather not update the code to handle a specific case like this.